index.js

```diff
@@ -1 +1,99 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db');
+const cors = require("cors");
+const server = express();
+
+server.get('/', (req, res) => {
+    res.send('Hello from Express');
+});
+
+server.listen(8080, () => 
+console.log('Server running on http://localhost:8080')
+);
+
+server.get('/api/users', (req, res) => {
```

Comment:
good job on building out your endpoint and using the correct HTTP codes for the relevant situations.